en/Obsidian Sync/Obsidian Sync and third-party services.md

```diff
@@ -9,4 +9,33 @@ If you set up the same third-party service on multiple devices, such as your pho
 > [!important] Files On-Demand
 > Some cloud storage services, such as OneDrive, allow you to only download files when you use them and later remove them locally to free up space. Since the files are no longer available locally, Obsidian Sync believes they have been deleted and removes them from your remote vault.
 >
-> To use Obsidian Sync together with Files On-Demand and similar features, make sure to configure the service to always keep the files on the device.
\ No newline at end of file
+> To use Obsidian Sync together with Files On-Demand and similar features, make sure to configure the service to always keep the files on the device.
+
+# Removing your vault from a third-party sync service
+If you are using Obsidian Sync and realize that you've set up your vault in a folder synced by a third-party service (such as Dropbox, OneDrive, or iCloud Drive), you can use the following steps to move the vault to a safer location.
+
+## Desktop (macOS/Windows/Linux)
+First, try the "Move vault" feature in the [[Vault switcher]]. 
+1. Open the vault switcher (in the default theme, this is represented by a vault icon in the bottom-left of the window).
+2. Select the three dots next to the vault you wish to move. 
+3. Choose a new location for the vault on your file system.
+
+If that doesn't work (e.g., you're moving across drives), do the following. (This might look like a lot of steps, but it's really just "move your vault folder, then open it in Obsidian as a 'new' vault.")
+  
+1. Make backups. (E.g., copy your vault folder somewhere else that you won't be touching throughout the rest of this operation.)  
```

Comment:
Maybe we should put a "Back up your vault" callout under the `## Remove your vault ...` heading. Feels like a good thing to do regardless of the option they choose, and a good warning before they pick an option.